concourse/pipelines/guest-package-build.yaml

```diff
@@ -285,6 +293,21 @@ jobs:
           universe: cloud-yum
           repo: google-guest-agent-el8
           environment: stable
+  - task: publish-time-since-stable
+    file: guest-test-infra/concourse/tasks/publish-time-since.yaml
+    vars:
+      last_publish_date: ((.:last-stable-date))
+      package_name: "google-guest-agent"
```

Comment:
This task is being added to the promote job for the package. This seems to imply that we would then only get stable staleness metric data points whenever a promotion is actually performed. Wouldn't this mean that we wouldn't be able to see something as being really stale until it is actually no longer stale (since we would have just run a promotion?)

An alternative approach to this would be a separate job which runs at a daily interval and does this calculation/metric publish directly for the package (rather than being coupled directly into the promotion to stable job).

Reply:
this metric only produces staleness data for each release, associated with each release

Reply:
Talked with @hopkiw directly -> this metric will be used to measure the specific releases staleness; we can set up another job to run daily to measure both image and package release staleness (for more adhoc understandings of staleness of the latest release).